### PR TITLE
feat: Translation to PT-BR (fixes #242)

### LINKED ISSUE #242: Translate to Brazilian Portuguese
### What language do you want to translate the website into?

Portuguese-BR

### What is the language code for the language?

pt-br

### What is the locale code for the language?

pt_BR

### What help will you need?

Help with the process after creating and commiting the file.

### Implementation Checklist

- [x] Create new site file
- [ ] Set up Netlify project
- [ ] Register new domain name
- [ ] Set up Google Analytics
- [ ] Set up Carbon Ads

### Participation

- [X] I am willing to submit a pull request for this change.

### Additional comments

_No response_

### PR DESCRIPTION
**Pull request by Rodzman**
Implementation of the Brazilian Portuguese translation.

- Added the `pt-br.yml` file with all the translations.
- Tested locally and had no issues.

@nzakas kindly review and feel free to make adjusment as needed.

### CODE REVIEW

**Comment by Rodzman** [comment]
Hi @nzakas I have a doubt about that EasyCLA validation. Whenever you're available, could you give me a hand here?

File: src/_data/sites/pt-br.yml [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+  name: Portuguese (BR)
+locale: pt-BR
+hostname: eslint.org
+
+#------------------------------------------------------------------------------
+# Analytics
+#------------------------------------------------------------------------------
+
+google_analytics:
+  code: "G-7DGPHY308T"
```
Comment (nzakas) [inline src/_data/sites/pt-br.yml:24]:
```suggestion
  code: "G-L4DVKBF1KH"
```

**Review by nzakas — COMMENTED** [review]
Thanks! I'll ask for some feedback from folks online, since I don't really know Portuguese. :)

File: src/_data/sites/pt-br.yml [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+#------------------------------------------------------------------------------
+# Global Settings
+#------------------------------------------------------------------------------
+
+language:
+  code: pt-br
+  flag: 🇧🇷
+  name: Portuguese (BR)
+locale: pt-BR
+hostname: eslint.org
```
[the remark this answers is not in the document]
Comment (nzakas) [reply src/_data/sites/pt-br.yml:17]: Shiny new URL for this translation.

File: src/_data/sites/pt-br.yml [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+google_analytics:
+  code: "G-7DGPHY308T"
+
+#------------------------------------------------------------------------------
+# Ads
+#------------------------------------------------------------------------------
+
+carbon_ads:
+  serve: CESDV2QM
+  placement: eslintorg
```
[the remark this answers is not in the document]
Comment (nzakas) [reply src/_data/sites/pt-br.yml:32]: We don't have an ad for this translation.

**Comment by Rodzman** [comment]
> Thanks! I'll ask for some feedback from folks online, since I don't really know Portuguese. :)

If you don't mind I've asked some of my brazilian friends to review it too. ;)

**Comment by nzakas** [comment]
Never too many eyes!

**Review by Lucas-D-Lima — APPROVED** [review]

**Review by chrisjullian — APPROVED** [review]

**Review by nzakas — APPROVED** [review]
Thanks! I'm going to get this set up now.

**Pull request merged**